test/index.spec.js

```diff
@@ -25,7 +26,9 @@ const pages = [
   '/pages/sunos/dd.md',
   '/pages/sunos/du.md',
   '/pages/sunos/svcs.md'
-];
+].map((x) => {
+  return path.join(x);
+});
```

Comment:
This is a bit strange. Doing a `path.join` on a single string makes no difference at all. What is the objective here?

Reply:
On windows, it converts the paths to use windows delimiters, so that for example, `/pages/sunos/svcs.md` becomes `\\pages\sunos\\svcs.md`. Without doing this, the tests in this suite fail as this is then used directly as mock data from methods not being tested (`utils.walk`).

Reply:
That's interesting. So even without passing individual parts of a file path, `path.join` splits them and reassembles them? If that is the case, then why are we passing individual strings in other cases? We can just wrap the string with a path.join.

Reply:
I think if anything, it would make sense to break each part of this array up into pieces, such that it's:

```js
const pages = [
  ['/index.json'],
  ['/pages', 'linux', 'apk.md'],
  ['/pages.zh', 'linux', 'apk.md'],
 // the rest of them
]
```

to avoid the confusion on what this does versus utilizing this trick elsewhere in the file.

Reply:
That makes sense to me. Let's do that.

Reply:
Hi @MasterOdin - just wanted to check on this.